src/Pester.ps1

```diff
@@ -253,6 +253,189 @@ function Has-Flag {
+
+    .PARAMETER ExcludePath
+
+    .PARAMETER ExcludeTagFilter
+
+    .PARAMETER FullNameFilter
+
+    .PARAMETER Output
+
+    .PARAMETER OutputFile
+    The path where Invoke-Pester will save formatted test results log file.
+    The path must include the location and name of the folder and file name with
+    the xml extension.
+    If this path is not provided, no log will be generated.
+
+    .PARAMETER OutputFormat
+    The format of output. Currently NUnitXml is supported.
+    Note that JUnitXml is not currently supported in Pester 5.
+
+    .PARAMETER PassThru
+    Returns a custom object (PSCustomObject) that contains the test results.
+    By default, Invoke-Pester writes to the host program, not to the output stream (stdout).
+    If you try to save the result in a variable, the variable is empty unless you
+    use the PassThru parameter.
+    To suppress the host output, use the Show parameter set to None.
+
+    .PARAMETER Path
+    Specifies a test to run. The value is a path\file
+    name or name pattern. Wildcards are permitted. All hash tables in a Script
+    parameter value must have a Path key.
+
+    .PARAMETER PesterOption
+    Sets advanced options for the test execution. Enter a PesterOption object,
+    such as one that you create by using the New-PesterOption cmdlet, or a hash table
+    in which the keys are option names and the values are option values.
+    For more information on the options available, see the help for New-PesterOption.
+
+    .PARAMETER Quiet
+    The parameter Quiet is deprecated since Pester v. 4.0 and will be deleted
+    in the next major version of Pester. Please use the parameter Show
+    with value 'None' instead.
+    The parameter Quiet suppresses the output that Pester writes to the host program,
+    including the result summary and CodeCoverage output.
+    This parameter does not affect the PassThru custom object or the XML output that
+    is written when you use the Output parameters.
+
+    .PARAMETER Show
+    Customizes the output Pester writes to the screen. Available options are None, Default,
+    Passed, Failed, Pending, Skipped, Inconclusive, Describe, Context, Summary, Header, All, Fails.
+    The options can be combined to define presets.
+    Common use cases are:
+    None - to write no output to the screen.
+    All - to write all available information (this is default option).
+    Fails - to write everything except Passed (but including Describes etc.).
+    A common setting is also Failed, Summary, to write only failed tests and test summary.
+    This parameter does not affect the PassThru custom object or the XML output that
+    is written when you use the Output parameters.
+
+    .PARAMETER Strict
+    Makes Pending and Skipped tests to Failed tests. Useful for continuous
+    integration where you need to make sure all tests passed.
+
+    .PARAMETER TagFilter
+
+    .EXAMPLE
```

Comment:
Add Pester 5 Examples.  A separate Invoke-PesterTests.ps1 script containing configuration settings would be helpful